f600233d1b154d507b782f84604bcb12/update/sys_script_include_9738ccda974611106fa8b4b3f153af13.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.getCssJwt</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>ATAT function to get a JWT from CSS to retreived information related to contracts.</description>
+        <name>getCssJwt</name>
+        <script><![CDATA[function getCssJwt(endpoint) {
+  var errClient = new ErrorHandler();
+  var jwtClient = new Jwt();
+  var reqClient = new Request();
+  var reqProps = {
+	endpoint: endpoint,
+	method: reqClient.POST,
+	mutualAuth: reqClient.JKS,
+  };
+  var ATAT_CONFIG_TABLE = "x_g_dis_atat_configuration";
+  var currentCssJwt = new GlideRecord(ATAT_CONFIG_TABLE);
+  var previousCssJwt = new GlideRecord(ATAT_CONFIG_TABLE);
+  var response;
+
+  try {
+    previousCssJwt.addQuery("name", "cssJwt");
+    previousCssJwt.query();
+
+    if (!previousCssJwt.next()) {
```

Comment:
Could cut down a tiny bit on the copypasta by placing the 5 lines that make the call and handle errors into a function

Reply:
Added `Request.getSuccessfulResponse` that returns a successful response or throws an error. Also added a `gs.debug("External request successful. Endpoint: " + reqProps.endpoint);` in `getSuccesssfulResponse` to provide the endpoint if the request is a success or failure, which you suggested in your comment above.